autotest.pm

```diff
@@ -295,33 +295,6 @@ sub start_process {
     return ($process, $child);
 }
 
-
-# TODO: define use case and reintegrate
-sub prestart_hook {
```

Comment:
Aren't these functions strictly a part of the API? It looks like they are not used by our test distribution and likely also not elsewhere, though.

Reply:
I think we only consider testapi.pm and *maybe* distribution.pm as part of the official, public API that should use deprecation before removal

Reply:
right